internal/nri/config.go

```diff
@@ -0,0 +1,38 @@
+package nri
+
+import (
+	nri "github.com/containerd/nri/pkg/adaptation"
+)
+
+// Config data for NRI.
+type Config struct {
+	Disable    bool   `toml:"disable"`
+	ConfigPath string `toml:"config_file"`
+	SocketPath string `toml:"socket_path"`
+	PluginPath string `toml:"plugin_path"`
+}
```

Comment:
This mostly duplicates internal/config/nri/config. Perhaps you can reuse that one by embedding it here (or vice versa).

Reply:
Simplified, eliminating internal/nri.Config altogether and switching to use directly internal/config/nri.Config instead.